data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -223,6 +223,24 @@
     "Michigan EJSCREEN Priority Community"
 )
 
+# CDC SVI INDEX percentile fields
+CDC_SVI_INDEX_SE_THEME_FIELD: str = "Socioeconomic Index"
+CDC_SVI_INDEX_HOUSEHOLD_THEME_COMPOSITION_FIELD: str = (
+    "Household Composition Index"
+)
+CDC_SVI_INDEX_LANGUAGE_THEME_FIELD: str = (
+    "Minority Status/Language Index"
+)
+CDC_SVI_INDEX_HOUSING_TRANSPORTATION_FIELD: str = (
+    "Housing Type/Transportation Index"
+)
+CDC_SVI_INDEX_RPL_THEMES_OVERALL_FIELD: str = (
+    "Overall rank for SVI indices"
+)
+CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY: str = (
+    "At or above 90 for overall percentile ranking according to SVI Index"
```

Comment:
In general we prefer to avoid acronyms, especially on "important" fields, so I'd suggest spelling out SVI as Social Vulnerability Index.

Reply:
I will address this for clarity